.github/workflows/code_review.yml

```diff
@@ -12,9 +12,10 @@
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: ./
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-          OPENAI_API_MODEL: "gpt-4-1106-preview"
+          INFERENCE_API_KEY: ${{ secrets.INFERENCE_API_KEY }}
+          MODEL: "gpt-4o-mini"
```

Comment:
The model name `gpt-4o-mini` should be verified for correctness and compatibility with the inference API being used. Ensure that this model is supported and that any necessary adjustments in the code or configuration are made.